src/components/modules/blockManager.ts

```diff
@@ -455,7 +455,7 @@ export default class BlockManager extends Module {
    * @returns {Block}
    */
   public getBlock(element: HTMLElement): Block {
-    if (!$.isElement(element)) {
```

Comment:
Not understand why you do this. Then i change return type isElement method, i start get errors
`[tsl] ERROR in /home/dimensi/projects/@editorjs/editor.js/src/components/modules/blockManager.ts(459,25)
      TS2339: Property 'parentNode' does not exist on type 'never'.`